### PR TITLE
Feature/add on

### PR DESCRIPTION
**Pull request by k-cao**
- separated documentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ install:
   - 'bower update'
 
 before_script:
-  - grunt # this does syntax checking, unit testing, and jshint
+  - grunt --force # this does syntax checking, unit testing, and jshint
```
Comment (denis-yuen) [inline .travis.yml]: Do not add a `--force` here. This may stop grunt from properly reporting syntax or semantic errors.

Comment (k-cao) [inline .travis.yml]: Not sure how to do this without force. It always says 'Toc' not initialized, but that's how the documentation does it. https://afeld.github.io/bootstrap-toc/#via-javascript

Comment (k-cao) [inline .travis.yml]: Also develop's branch travis.yml also has grunt --force

Comment (k-cao) [inline .travis.yml]: Okay I turned it grunt warning about Toc only off with  /*global Toc*/

**Pull request closed without merging**